extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/FastBootHibernatePersistenceProvider.java

```diff
@@ -381,7 +381,7 @@ private static void injectDataSource(String persistenceUnitName, String dataSour
     private static void injectRuntimeConfiguration(HibernateOrmRuntimeConfigPersistenceUnit persistenceUnitConfig,
             Builder runtimeSettingsBuilder) {
         // Database
-        runtimeSettingsBuilder.put(AvailableSettings.HBM2DDL_DATABASE_ACTION,
+        runtimeSettingsBuilder.put(AvailableSettings.JAKARTA_HBM2DDL_DATABASE_ACTION,
```

Comment:
Interestingly, changing the property we use here leads to a failure in `UnsupportedPropertiesTest` .That test tries to override the `javax.` property and expects a warning because you can't override properties set by Quarkus.

Now that we're no longer using the `javax.` property, there no longer is a warning.

The short term solution is to update the test to use the `jakarta` property, but I wonder: shouldn't we also prevent the use of `javax` properties in `quarkus.hibernate-orm.unsupported-properties`, or at least warn against their use, because of that kind of problems? Same with `persistence.xml`.

Reply:
Good point, and thanks for digging :)
I'll update the test and open a follow-up issue to improve the validations.

Reply:
rebased; follow up is here: https://github.com/quarkusio/quarkus/issues/31130